machines/vcItem.ts

```diff
@@ -1003,16 +1007,26 @@ export const vcItemMachine =
           { to: (context) => context.serviceRefs.store }
         ),
 
-        loadMyVcs: send(StoreEvents.GET(MY_VCS_STORE_KEY), {
-          to: (context) => context.serviceRefs.store,
-        }),
-
         removeVcItem: send(
           (_context, event) => {
             return StoreEvents.REMOVE(MY_VCS_STORE_KEY, _context.vcKey);
           },
           { to: (context) => context.serviceRefs.store }
         ),
+
+        storingActivityLog: send(
+          (context, _) =>
+            ActivityLogEvents.LOG_ACTIVITY({
+              _vcKey: '',
+              type: 'VC_REMOVED',
+              timestamp: Date.now(),
+              deviceName: '',
+              vcLabel: String(context.id),
```

Comment:
context.id will defaultly gives you id in string only, we don't have to type cast again.